docs/data/toolpad/how-to-guides/connect-to-databases.md

```diff
@@ -14,73 +15,77 @@ Inside `/resources/functions.ts`, we can create a custom function:
 import mysql from 'mysql2/promise';
 import fs from 'fs';
 
-async connectionFn () {
-   const connection = await mysql.createConnection({
-   host: process.env.MYSQL_HOST,
-   port: process.env.MYSQL_PORT,
-   user: process.env.MYSQL_USERNAME,
-   password: process.env.MYSQL_PASSWORD,
-   database: process.env.MYSQL_DATABASE
-   });
-   return connection;
+async function createConnection() {
+  const connection = await mysql.createConnection({
+    host: process.env.MYSQL_HOST,
+    port: process.env.MYSQL_PORT,
+    user: process.env.MYSQL_USERNAME,
+    password: process.env.MYSQL_PASSWORD,
+    database: process.env.MYSQL_DATABASE,
+  });
+  return connection;
 }
 
 export async function getData(order_id: number) {
-   const query = `SELECT * FROM table WHERE order_id=${order_id}`;
```

Comment:
As a developer who would read the docs, this would be a red flag for me (= I'm not using that)

Reply:
We heavily penalize submissions when this appears in the fullstack challenge